components/brave_rewards/resources/extension/brave_rewards/_locales/en_US/messages.json

```diff
@@ -185,6 +185,10 @@
     "message": "You have a grant waiting for you.",
     "description": "Description for new grant notification"
   },
+  "backupNow": {
```

Comment:
you need to add `backupNow` to `locale_api.ts` as well. Current view
![image](https://user-images.githubusercontent.com/9574457/52941566-6de7c800-3369-11e9-8a4f-c791878cc0ec.png)

Reply:
done